build: local build fails

Using `npm install` fails the build because of a missing dependency for `run-p`. I've then installed https://www.npmjs.com/package/npm-run-all globally only to run into the next error.

I am using Windows 11, npm 8.11.0, node 16.16.0

````
C:\Daten\Soren\static-cms-hugo-netlify-template>npm install
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm ERR! code 1
npm ERR! path C:\Daten\Soren\static-cms-hugo-netlify-template\node_modules\hugo-bin
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c rimraf vendor && node lib/install.js
npm ERR! node:internal/process/esm_loader:94
npm ERR!     internalBinding('errors').triggerUncaughtException(
npm ERR!                               ^
npm ERR!
npm ERR! Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'C:\Daten\Sören\static-cms-hugo-netlify-template\node_modules\pkg-conf\node_modules\find-up\' imported from C:\Daten\Sören\static-cms-hugo-netlify-template\node_modules\pkg-conf\index.js
npm ERR! Did you mean to import find-up/index.js?
npm ERR!     at new NodeError (node:internal/errors:372:5)
npm ERR!     at legacyMainResolve (node:internal/modules/esm/resolve:341:9)
npm ERR!     at packageResolve (node:internal/modules/esm/resolve:941:14)
npm ERR!     at moduleResolve (node:internal/modules/esm/resolve:1003:20)
npm ERR!     at defaultResolve (node:internal/modules/esm/resolve:1218:11)
npm ERR!     at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
npm ERR!     at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
npm ERR!     at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
npm ERR!     at link (node:internal/modules/esm/module_job:78:36) {
npm ERR!   code: 'ERR_MODULE_NOT_FOUND'
npm ERR! }

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Soren\AppData\Local\npm-cache\_logs\2023-01-17T18_44_29_259Z-debug-0.log
````

--- a/webpack.common.js
+++ b/webpack.common.js
@@ -29,7 +29,10 @@ module.exports = {
         loader: "babel-loader",
         test: /\.js?$/,
         exclude: /node_modules/,
-        options: {cacheDirectory: true}
+        options: {
+          cacheDirectory: true,
+          presets: ['@babel/preset-env','@babel/preset-react']
+        }
       },
       {
         test: /\.(sa|sc|c)ss$/,

